js/defaults_dialog.js

```diff
@@ -229,6 +229,30 @@ helper.defaultsDialog = (function() {
                     key: "small_angle",
                     value: 180
                 },
+                {
+                    key: "nav_fw_control_smoothness",
+                    value: 2
+                },
+                {
+                    key: "failsafe_procedure",
+                    value: "RTH"
+                },
+                {
+                    key: "nav_rth_allow_landing",
+                    value: "NEVER"
```

Comment:
I'd say it's slightly problematic since in case of not-recoverable failsafe model would circle  above takeoff until battery dies and model uncontrollably descend in random direction with random speed

Reply:
This is not really what I have observed. Whenever I hear of this, the model's motor stops, and the plane glides down in the same loiter pattern that it was flying in. Though I know this is not intentional, it does seem to "land" as well as the auto land feature. It's just unpowered.

Another thing to consider is that not all flying sites are suitable for auto land, which requires a large, open space with minimal obstacles. Which is why implementing https://github.com/iNavFlight/inav/issues/6173 would be great.

```suggestion
                    value: "FS_ONLY"
```

Reply:
FS_ONLY is a better option indeed

Reply:
During brown-out everything is possible. You have no guarantee what dies first and in what state servos will be as well

Reply:
Agree, especially with 2S setups. With 3S and above there is usually enough energy to still power the fc and servos. Not always the case though.